core/src/syn/test.rs

```diff
@@ -9,7 +9,7 @@ use crate::syn::token::{t, TokenKind};
 
 impl Parse<Self> for Value {
 	fn parse(val: &str) -> Self {
-		super::value(val).unwrap()
+		super::value_field(val).inspect_err(|e| eprintln!("{e}")).unwrap()
```

Comment:
I guess we want to keep this: `eprintln!`?

Reply:
I put that there during debugging but I left it in as it was useful to have and that function is only called in tests. It makes the error a bit nicer to work with.